lib/src/main/java/com/thenewboston/api/bankapi/repository/BankRepository.kt

```diff
@@ -24,40 +24,41 @@ import com.thenewboston.data.dto.bankapi.upgradenoticedto.UpgradeNoticeRequest
 import com.thenewboston.data.dto.bankapi.validatorconfirmationservicesdto.ConfirmationServicesList
 import com.thenewboston.data.dto.bankapi.validatorconfirmationservicesdto.request.PostConfirmationServicesRequest
 import com.thenewboston.data.dto.bankapi.validatordto.ValidatorList
+import com.thenewboston.utils.PaginationOptions
 import io.ktor.util.*
 import javax.inject.Inject
 
 @KtorExperimentalAPI
 class BankRepository @Inject constructor(private val dataSource: BankDataSource) {
 
-    suspend fun banks(): Outcome<BankList> = dataSource.fetchBanks()
+    suspend fun banks(): Outcome<BankList> = dataSource.fetchBanks(PaginationOptions())
```

Comment:
What is BankRepository used for and will it require Pagination?
I ask because it will determine if I add a pagination parameter to its methods and pass it to the `dataSource.fetch` methods

Like so:
```
suspend fun banks( pagination: PaginationOptions()): Outcome<BankList> = dataSource.fetchBanks(pagination)

```

Reply:
Repository is a pattern from clean architecture. It exposes the functions from the data source thus hides the business logic. This is what the SDK user will be calling and it should not call data source directly.

In this case, we don't need to introduce a new instance here.

Reply:
Ok, Thank you for Clearing that up.
Is there anything else you would like me to modify in the code?

Reply:
No, this should be the last review I promise! 😀

Reply:
Hey did you remove the null constructor in favour of passing in the data class as a parameter? Once that's done, we can approve and get it merged.